src/Attributes/Operation.php

```diff
@@ -12,12 +14,31 @@ class Operation
     /** @var array<string> */
     public array $tags;
 
+    public ?string $security;
+
     public ?string $method;
 
-    public function __construct(string $id = null, array $tags = [], string $method = null)
+    /**
+     * @param string|null $id
+     * @param array $tags
+     * @param \Vyuldashev\LaravelOpenApi\Factories\SecuritySchemeFactory|string|null $security
+     * @param string|null $method
+     * @throws InvalidArgumentException
+     */
+    public function __construct(string $id = null, array $tags = [], string $security = null, string $method = null)
     {
         $this->id = $id;
         $this->tags = $tags;
         $this->method = $method;
+
+        if ($security) {
+            $this->security = class_exists($security) ? $security : app()->getNamespace().'OpenApi\\SecuritySchemes\\'.$security.'SecurityScheme';
```

Comment:
I'm not sure if I did it right by hardcoding `'SecurityScheme'`.

Reply:
Please, change `'.$security.'SecurityScheme';` to `'.$security;`